static/sass/_patterns_card.scss

```diff
@@ -0,0 +1,86 @@
+@mixin snapcraft-p-card {
+
+  .p-card__category {
```

Comment:
This file (and `u-crop` styles as well) seem like they were possibly copied from other project (insights blog?) - if so it would be good to comment with a link to a file copied for reference.